src/STRPAK.H

```diff
@@ -28,12 +28,7 @@ const int STRPPEXT   = 8;
 inline int fcn##W( int c) { return fcn( c); } \
 inline int fcn##W( char c) { return fcn( (unsigned char)c); } \
 inline int fcn##W( unsigned char c) { return fcn( (unsigned char)c); }
-// cause error in VS2008 (won't compile in VS6 due to Dundas grid STL troubles, 5-31-06)
-#if _MSC_VER >= 1400
 #define charTypeErr( fcn) Should_use_##fcn##W
```

Comment:
I think we should add a comment here to help avoid confusion in the future. Something like, "`fcn##W is needed to handle type conversions that the original functions get tripped up on."

Reply:
The idea here is to explain what's happening on line 33. I like the direction you are going, but I think it might be more accurate to say:

> `charTypeErr` is used to redefine the original functions so that when they are used, the compiler will throw an error because "Should_use_##fcn##W" is not defined. This indicates to the user that they should use the "W" suffixed version of the function instead for proper casting.